### PR TITLE
fix: Daily Habits Dual Rendering Bug

### PR DESCRIPTION
**Pull request by calebpanza**
## ✏️ Description
When moving through the different steps of the Daily Habit, different areas of the Daily Habit Tab would render the previous _and_ new state overtop each other.

## 🔬 To Test
1. Run through the Daily Habits

## ⚠️ To-do Before Merge
n/a

## 📝 Checklist

- [x] I have performed a self-review of my code
- [x] I have tested my code works in the project (nothing breaks)
- [x] I have commented my code in hard-to-understand areas
- [x] New and existing tests and linting pass locally with my changes
- [x] I have made corresponding changes to the documentation (new envs?)
- [x] For PR's with design updates/changes, I have consulted with the product designer and made all necessary adjustments in Figma

### CODE REVIEW

File: templates/mobile/ios/apolloschurchapp.xcworkspace/xcshareddata/xcschemes/apolloschurchapp.xcscheme [hunk 1]
```diff
@@ -99,7 +99,7 @@
       </BuildableProductRunnable>
    </LaunchAction>
    <ProfileAction
-      buildConfiguration = "Release"
```
Comment (vinnyjth) [inline templates/mobile/ios/apolloschurchapp.xcworkspace/xcshareddata/xcschemes/apolloschurchapp.xcscheme:102]: this change needed?

File: templates/mobile/ios/apolloschurchapp.xcworkspace/xcshareddata/xcschemes/apolloschurchapp.xcscheme [hunk 2]
```diff
@@ -99,7 +99,7 @@
...
+      buildConfiguration = "Debug"
```
Comment (redreceipt) [inline templates/mobile/ios/apolloschurchapp.xcworkspace/xcshareddata/xcschemes/apolloschurchapp.xcscheme:102]: why this change?

**Comment by calebpanza** [comment]
@redreceipt @vinnyjth I accidentally committed a change to the iOS Scheme in a previous PR that changed the build configuration from "Debug" to "Release". I also reverted that change, here, since doing that was _only_ suppose to be done in order to test this bug

**Comment by vinnyjth** [comment]
@calebpanza link me to that PR and I'll approve :)

**Review by redreceipt — APPROVED** [review]

**Comment by calebpanza** [comment]
Nevermind

**Comment by calebpanza** [comment]
I had changed Run, not Profile, and it looks like that got reverted at some point. Sorry! So many changes and I lost track of that

**Review by dsilva609 — APPROVED** [review]
